bot/exts/evergreen/trivia_quiz.py

```diff
@@ -163,7 +386,7 @@ def check(m: discord.Message) -> bool:
                     break
 
                 # Reduce points by 25 for every hint/time alert that has been sent.
```

Comment:
This comment also explains exactly what the code says, so the comment is not needed

Reply:
resolved